tests/customer-resource-show-visibility-test.js

```diff
@@ -69,25 +76,104 @@ describeApplication('CustomerResourceShowVisibility', () => {
       });
     });
 
-    it('displays the hidden/reason section', () => {
-      expect(CustomerResourceShowPage.isHidden).to.be.true;
+    it('the hide/show toggle with hide', () => {
+      expect(CustomerResourceShowPage.isHidden).to.be.false;
     });
 
     it('maps the hidden reason text', () => {
       expect(CustomerResourceShowPage.hiddenReason).to.equal('Set by System');
     });
   });
 
-  describe('visiting the customer resource show page with a hidden resource and reason text is not mapped', () => {
+  describe('visiting the customer resource show page with a hidden resource and no reason', () => {
+    beforeEach(function () {
+      resource = this.server.create('customer-resource', 'isHidden', {
+        package: pkg,
+        isSelected: true,
+        title
+      });
+
+      let visibilityData = this.server.create('visibility-data', {
+        isHidden: true,
+        reason: ''
+      }).toJSON();
+
+      resource.update('visibilityData', visibilityData);
+      resource.save();
+
+      return this.visit(`/eholdings/customer-resources/${resource.id}`, () => {
+        expect(CustomerResourceShowPage.$root).to.exist;
+      });
+    });
+
+    it('the hide/show toggle with hide', () => {
+      expect(CustomerResourceShowPage.isHidden).to.be.false;
+    });
+
+    it('maps the hidden reason text', () => {
+      expect(CustomerResourceShowPage.hiddenReason).to.equal('');
+    });
+  });
+
+  describe('visiting the customer resource show page and hiding a resource', () => {
+    beforeEach(function () {
+      resource = this.server.create('customer-resource', {
+        package: pkg,
+        isSelected: true,
+        title
+      });
+
+      return this.visit(`/eholdings/customer-resources/${resource.id}`, () => {
+        expect(CustomerResourceShowPage.$root).to.exist;
+      });
+    });
+
+    it('displays the hide/show toggle with show', () => {
```

Comment:
I think this label is reversed? `displays the visibility toggle as switched to on` would work.